examples/plotting/polar.py

```diff
@@ -1,3 +1,11 @@
+'''This example shows how to use polar coordinates to generate a grid of polar plots of rose/rohdonea curves.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.colors.named, bokeh.io.show, bokeh.models.ColumnDataSource, bokeh.models.Plot, bokeh.models.PolarTransform, bokeh.plotting.figure, bokeh.plotting.figure.line, bokeh.plotting.gridplot
```

Comment:
```suggestion
    :apis: bokeh.colors.named, bokeh.io.show, bokeh.models.ColumnDataSource, bokeh.models.Plot, bokeh.models.PolarTransform, bokeh.plotting.figure, bokeh.plotting.figure.line, bokeh.plotting.gridplot # noqa: E501
```
This line violates the max line length of 165 letters. To silence this warning, you can add `# noqa: E501` to the end of the line or of the whole code block.

But I would suggest to remove 2 elements
```suggestion
    :apis: bokeh.io.show, bokeh.models.ColumnDataSource, bokeh.models.PolarTransform, bokeh.plotting.figure, bokeh.plotting.figure.line, bokeh.plotting.gridplot
```

Reply:
> This line violates the max line length of 165 letters. To silence this warning, you can add `# noqa: E501` to the end of the line or of the whole code block.
> 
> But I would suggest to remove 2 elements

Thanks!, as you've suggested, I've removed these two elements + whitespace from earlier !